DESCRIPTION

```diff
@@ -15,7 +15,8 @@ Imports:
     RJSONIO,
     packrat (>= 0.4.2-1),
     yaml (>= 2.1.5),
-    rstudioapi (>= 0.2)
+    rstudioapi (>= 0.2),
+    rmarkdown (>= 0.5.2)
```

Comment:
Note this means we depend on the embedded version of rmarkdown so we also needed to do this: https://github.com/rstudio/rstudio/commit/db3633f7ea63098d91e532c5cf513ff954df05f5 (we can remove this once we depend on rmarkdown from CRAN)